invokeai/backend/model_patcher.py

```diff
@@ -107,6 +94,7 @@ def apply_lora(
         model: AnyModel,
         loras: Iterator[Tuple[LoRAModelRaw, float]],
         prefix: str,
+        model_state_dict: Optional[Dict[str, torch.Tensor]] = None,
```

Comment:
Can you add a docstring explaining this new `model_state_dict` parameter?

Reply:
Done.